tests/rundb/test_httpdb.py

```diff
@@ -747,3 +750,65 @@ def _assert_projects(expected_project, project):
     )
     assert expected_project.spec.desired_state == project.spec.desired_state
     assert expected_project.spec.desired_state == project.status.state
+
+
+def test_watch_logs_continue():
+
+    # create logs with invalid utf-8 byte
+    log_lines = [
+        b"Firstrow",
+        b"Secondrow",
+        b"Thirdrow",
+        b"Smiley\xf0\x9f\x98\x86",
+        b"\xf0",  # invalid utf-8 - should be replaced with U+FFFD (�)
+        b"LastRow",
+    ]
+    log_contents = b"".join(line for line in log_lines)
+    db = mlrun.db.httpdb.HTTPRunDB("http+mock://wherever.com")
+    run_uid = "some-uid"
+    project = "some-project"
+    adapter = requests_mock.Adapter()
+    current_log_line = 0
+
+    # assert that the log contents are invalid utf-8
+    with pytest.raises(UnicodeDecodeError):
+        for log_line in log_lines:
+            log_line.decode()
+
+    def callback(request, context):
+        nonlocal current_log_line
+        offset = int(request.qs["offset"][0])
+        current_log_line += 1
+
+        # when offset is 0 -> return first log line
+        # when offset is len(log_lines[i]) -> return second log line, and so on
+        # the idea is to always return the next log line, extracted by the log contents
+        # to test offset calculation is valid from the client set
+        context.status_code = 200
+        if current_log_line < len(log_lines):
+            context.headers["x-mlrun-run-state"] = "running"
+        len_next_word = len(log_lines[current_log_line - 1])
+        contents = log_contents[offset : offset + len_next_word]
+        return contents
+
+    adapter.register_uri(
+        "GET",
+        f"http+mock://wherever.com/api/v1/log/{project}/{run_uid}",
+        content=callback,
+    )
+    db.session = db._init_session()
+    db.session.mount("http+mock", adapter)
+    mlrun.mlconf.httpdb.logs.pull_logs_default_interval = 0.1
+    with unittest.mock.patch("sys.stdout", new_callable=io.StringIO) as newprint:
+        db.watch_log(run_uid, project=project)
+        assert (
+            newprint.getvalue()
+            == log_contents[0 : len(log_lines[0])].decode(errors="replace")
```

Comment:
```suggestion
            == log_contents[0 : len(log_lines[0])].decode(errors=mlrun.mlconf.httpdb.logs.decode.errors)
```
Unless we assume the config mode will always be `replace` in which case there's no point in putting it in a config parameter.